install/Knative-with-AKS.md

```diff
@@ -142,46 +142,24 @@ rerun the command to see the current status.
 > Note: Instead of rerunning the command, you can add `--watch` to the above
   command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
 
-## Installing Knative components
+## Installing Knative
 
-You can install the Knative Serving and Build components together, or Build on its own.
-
-### Installing Knative Serving and Build components
+The following commands install all available Knative components. To customize
+your Knative installation, see [Performing a Custom Knative Installation](Knative-custom-install.md).
```

Comment:
So you will add a new file `Knative-custom-install.md`?

Reply:
That page is being added in #542, which is why I said in the description of this PR that that must be merged in first.